src/include/columnar/columnar_tableam.h

```diff
@@ -51,6 +49,9 @@ typedef struct ColumnarScanDescData *ColumnarScanDesc;
 const TableAmRoutine * GetColumnarTableAmRoutine(void);
 extern void columnar_tableam_init(void);
 extern void columnar_tableam_finish(void);
+extern void CreateTruncateTrigger(Oid relationId);
+extern void EnsureTableOwner(Oid relationId);
+extern bool CheckCitusVersion(int elevel);
```

Comment:
These two are defined in the `distributed/` directory, and referenced in the `citus/` directory, so they seem to be dependencies that we need to fix.